code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -105,6 +106,11 @@
 		/obj/item/weapon/reagent_containers/food/snacks/meat/animal/monkey = 4,
 	)
 
+/obj/structure/closet/secure_closet/freezer/meat/get_heat_conductivity()
+	return HEAT_CONDUCTIVITY_REFRIGERATOR
+
+/obj/structure/closet/secure_closet/freezer/fridge/get_heat_conductivity()
+	return HEAT_CONDUCTIVITY_REFRIGERATOR
```

Comment:
why not define this on the parent `/obj/structure/closet/secure_closet/freezer`

Reply:
Kitchen Cabinet is the base type `/obj/structure/closet/secure_closet/freezer` but I figured it didn't seem like it should be as  insulated as the meat fridge since it normally stores just flour and sugar.

Reply:
and eggs

Reply:
i mean, the base name is "freezer", those are insulated

Reply:
Actually.. yeah since all `/obj/structure/closet/secure_closet/freezer` cool down their contents, I moved it to the base type.